be/src/olap/key_coder.h

```diff
@@ -221,4 +221,13 @@ class KeyCoderTraits<OLAP_FIELD_TYPE_VARCHAR> {
     }
 };
 
+struct EnumClassHash
```

Comment:
1. why not put this in cpp file?
2. we prefer `struct EnumClassHash {`

Reply:
done